apps/server/src/modules/user-login-migration/loggable/user-is-already-migrated.loggable-exception.ts

```diff
@@ -0,0 +1,27 @@
+import { HttpStatus } from '@nestjs/common';
+import { BusinessError } from '@shared/common';
+import { ErrorLogMessage, Loggable, LogMessage, ValidationErrorLogMessage } from '@src/core/logger';
+
+export class MultipleUsersFoundLoggableException extends BusinessError implements Loggable {
+	constructor() {
+		super(
+			{
+				type: 'MULTIPLE_USERS_MIGRATION_LOGGABLE_EXCEPTION',
+				title: 'User has already migrated',
+				defaultMessage: 'User has already migrated',
+			},
+			HttpStatus.INTERNAL_SERVER_ERROR,
+			{
+				multipleUsersFound: true,
```

Comment:
Why this? Seems that you want to use the same loggable expection for both error messages right? would be better to have different one